api/types.proto

```diff
@@ -624,6 +624,15 @@ message RaftConfig {
 	uint32 election_tick = 5;
 }
 
+
+// SecretsConfig specifies cluster-level secrets settings.
+message SecretsConfig {
+	// MaxSecrets specifies how many user-specified secrets there can be
+	uint32 max_secrets = 1;
+	// MaxSecretVersions specifies how many versions of a secret there can be
+	uint32 max_secret_versions = 2;
```

Comment:
Will `0` values be interpreted as implying the default values? I'm thinking about the situation where an existing cluster is upgraded to a version that supports secrets. These variables will equal `0` in that case since the cluster object was not created with a `Secrets` config.